addons/sale_pdf_quote_builder/wizards/res_config_settings_views.xml

```diff
@@ -16,6 +16,10 @@
                         <field name="sale_header_name" invisible="1"/>
                         <label for="sale_header" class="me-2"/>
                         <field name="sale_header" filename="sale_header_name" options="{'accepted_file_extensions': '.pdf'}"/>
+                        <!-- TODO vava: sale_header undefined... -->
+                        <a invisible="not sale_header" t-attf-href="/web/content/?model=ir.attachment&amp;id={{sale_header}}&amp;field=datas&amp;download=true" class="oe_link" target="_blank">
+                            <i class="fa fa-download"/> Download
+                        </a>
```

Comment:
IMHO, you won't be able to make it work unless you make a custom js widget for that (or improve the standard one for binary fields).